aiida_firecrest/utils_test.py

```diff
@@ -47,10 +48,15 @@ def __init__(
         self._scratch = tmpdir / "scratch"
         self._scratch.mkdir()
         self._client_id = "test_client_id"
-        self._client_secret = "test_client_secret"
+
+        Path(tmpdir / ".firecrest").mkdir()
```

Comment:
pytest has utilities for this, please consider this, because it automatically cleans up the tmp folder for you. This usually makes tests more robust. I think this test fails if you run it twice as in the second run the directory already exists

https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
I kept it there, because `_create_secret_file()` should be able to handle all cases, if there is a directory already, if there is a secret file inside already, etc..

Reply:
The mkdir will fail if you create the path two times. It works because it is used actually already with the fixture in the `conftest.py` (did not see this before). I find a bit confusing that test utils are in the source code, but lets not change this for now^^

Reply:
This file is removed from interface now. The new tests only relies on `tests/conftest.py`